app/views/menu/help-menu.tmpl.html

```diff
@@ -0,0 +1,21 @@
+<span role="help-button">
```

Comment:
BTW, all of our current menu items following that structure - therefore I would leave it.